### PR TITLE
add path_save_dir argument in the example prediction script

### PR DESCRIPTION
**Pull request by hsiaoyi0504**
@alxndrkalinin Correct me if I am wrong that It seems to me that we need this option to generate the output into a folder.

### CODE REVIEW

**Comment by alxndrkalinin** [comment]
I don't think this is necessary, since `path_save_dir` already [defaults to `predictions`](https://github.com/alxndrkalinin/pytorch_fnet/blob/92236e2b55b13813fd1eca63df3f48f8742f40e4/fnet/cli/predict.py#L301) in `fnet/cli/predict.py`.
If you want to be able to change the output location, you could define an argument. But again, that'd be redundant, as it just repeats the API of `fnet/cli/predict.py`, which I recommend using for running predictions anyway.

**Comment by hsiaoyi0504** [comment]
@alxndrkalinin I know it's not necessary. I am thinking adding this might help someone get started in the future, though.

**Comment by alxndrkalinin** [comment]
then you should add it as a command line argument

**Comment by hsiaoyi0504** [comment]
@alxndrkalinin I didn't get it. I did add it as a command-line argument.

File: examples/predict.py [hunk 1]
```diff
@@ -18,13 +18,15 @@
 model_save_dir = "{}/model/".format(os.getcwd())
 
 data_save_path_test = "{}/image_list_test.csv".format(os.getcwd())
+path_save_dir = "{}/predictions/".format(os.getcwd())
```
Comment (alxndrkalinin) [inline examples/predict.py:21]: this line creates a string variable containing current path, appended by `/predictions/`

File: examples/predict.py [hunk 2]
```diff
@@ -18,13 +18,15 @@
...
 
 command_str = (
     "python ../fnet/cli/predict.py "
     "--path_model_dir {} "
     "--dataset fnet.data.MultiChTiffDataset "
     '--dataset_kwargs \'{{"path_csv": "{}"}}\' '
-    "--gpu_ids {}".format(model_save_dir, data_save_path_test, gpu_id)
+    "--path_save_dir {} "
```
Comment (alxndrkalinin) [inline examples/predict.py:28]: this line incorporates the new variable into the string of parameters that will be used to call `../fnet/cli/predict.py` script

**Comment by alxndrkalinin** [comment]
I left comments toy your changes, see above. What is happening there is that the value of `path_save_dir` variable is defined as `/predictions/` subfolder and then passed to the `../fnet/cli/predict.py` script. This change doesn't really make difference because:
* on one hand `../fnet/cli/predict.py` already will save predictions to the `/predictions/` subfolder (since it's already the default value if we don't include `path_save_dir` parameter)
* on the other, the value of the variable you introduced is hardcoded and cannot be redefined, because you don't add it to the command line argument parsing (like it's done for the `gpu_id` variable). 
That's what I mean by adding the command line parameter.

**Comment by hsiaoyi0504** [comment]
Got it. That makes sense. I will update it then. I originally thought you mean the command line parameters of `../fnet/cli/predict.py` and didn't think of that you mean the command line parameters of `examples/predict.py`.

**Pull request closed without merging**